### PR TITLE
Statement page nav

### PR DESCRIPTION
**Pull request by will0684**
Added navigation to statement page, small css changes

### CODE REVIEW

**Comment by will0684** [comment]
On it!

**Comment by will0684** [comment]
Tests are added and working!

**Comment by will0684** [comment]
Have to get the unit test for NavParams working properly, as it is right now it's not working exactly as it should, will get on that as next task

**Comment by will0684** [comment]
Alright, ready for your review Calvin or anyone else who feels up to it. Thanks.

File: src/pages/acceptance-statement/acceptance-statement.html [hunk 1]
```diff
@@ -31,8 +31,8 @@ <h1 class="statement-header">Acceptance Statement</h1>
     <p class="statement-content">I acknowledge that it is my responsibility to read and understand the information contained within each heading presented
```
Comment (CalvinRodo) [inline src/pages/acceptance-statement/acceptance-statement.html]: Add translations.

File: src/pages/acceptance-statement/acceptance-statement.html [hunk 2]
```diff
@@ -31,8 +31,8 @@ <h1 class="statement-header">Acceptance Statement</h1>
...
       on this page.</p>
     <ion-buttons>
-      <button class="accept-button" ion-button>I Accept</button>
-      <a class="refuse-button" href="">I refuse</a>
+      <button class="accept-button" ion-button (click)="acceptStatement(report)">I Accept</button>
```
Comment (CalvinRodo) [inline src/pages/acceptance-statement/acceptance-statement.html]: Add translations

**Comment by CalvinRodo** [comment]
Wheres the french text for this?

**Pull request merged**